nixos/modules/services/monitoring/karma.nix

```
@@ -117,7 +117,7 @@ in
         Type = "simple";
         DynamicUser = true;
         Restart = "on-failure";
-        ExecStart = "${pkgs.karma}/bin/karma --config.file ${cfg.configFile} ${lib.concatStringsSep " " cfg.extraOptions}";
+        ExecStart = "${lib.getExe cfg.package}/bin/karma --config.file ${cfg.configFile} ${lib.concatStringsSep " " cfg.extraOptions}";
```

lib.getExe should replace the explicit bin path, right? 

```suggestion
        ExecStart = "${lib.getExe cfg.package} --config.file ${cfg.configFile} ${lib.concatStringsSep " " cfg.extraOptions}";
```

---

Correct. Stupid oversight be me. Should be fixed now :)